Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -60,9 +65,123 @@ YUI.add('ez-contenteditviewservice', function (Y) {
                 service._loadContentType(resources.ContentType, tasks.add());
 
                 tasks.done(function () {
-                    next(service);
+                    service._setVersionFields(content, translationExists, function () {
```

Comment:
`_setVersionFields` is completely synchronous so it should not have a callback parameter and you can just write:
```js
service._setVersionFields(content, translationExists);
next(service);
```